TRANSPARENCY_FAQS.md

```diff
@@ -0,0 +1,64 @@
+# Semantic Kernel Responsible AI FAQs
+
+## What is Microsoft Semantic Kernel?
+Microsoft Semantic Kernel is a lightweight, open-source development kit designed to facilitate the integration of AI models into applications written in languages such as C#, Python, or Java.
+
+It serves as efficient middleware that supports developers in building AI agents, automating business processes, and connecting their code with the latest AI technologies. Input to this system can range from text data to structured commands, and it produces various outputs, including natural language responses, function calls, and other actionable data.
+
+
+## What can Microsoft Semantic Kernel do?
+Building upon its foundational capabilities, Microsoft Semantic Kernel facilitates several functionalities:
+-	AI Agent Development: Users can create agents capable of performing specific tasks or interactions based on user input.
+-	Function Invocation: It can automate code execution by calling functions based on AI model outputs.
+-	Modular and Extensible: Developers can enhance functionality through plugins and a variety of pre-built connectors, providing flexibility in integrating additional AI services.
+-	Multi-Modal Support: The kernel easily expands existing applications to support modalities like voice and video through its architecture
+
+
+## What is/are Microsoft Semantic Kernel’s intended use(s)?
+The intended uses of Microsoft Semantic Kernel include:
+- 	Enterprise Applications: Building scalable enterprise-grade solutions that can leverage advanced AI models for various business applications.
+-	Automation of Business Processes: Facilitating quick and efficient automation of workflows and tasks within organizations.
+- 	Integration of AI Services: Connecting client code with a variety of pre-built AI services and capabilities for rapid development.
+
+
+## How was Microsoft Semantic Kernel evaluated? What metrics are used to measure performance?
+Microsoft Semantic Kernel was reviewed for reliability and performance metrics that include:
+-	Accuracy: Evaluated based on the correctness of the outputs generated against known facts.
+-	Integration Speed: Assessed by the time taken to integrate AI models and initiate functional outputs based on telemetry.
+-	Performance Consistency: Measurements taken to verify the system's reliability based on telemetry.
+
+
+## What are the limitations of Microsoft Semantic Kernel?
+Limitations include:
+-	Contextual Misunderstanding: The system may struggle with nuanced requests, particularly those involving complex context.
+-	Bias in LLM Outputs: Historical biases in the training data can inadvertently influence model outputs. 
+	-	Users can mitigate these issues by:
+		-	Formulating clear and explicit queries.
+		-	Regularly reviewing AI-generated outputs to identify and rectify biases or inaccuracies.
+        -   Providing relevant information when prompting the LLM so that it can base it's responses on this data
```

Comment:
These limitations are really limitations of the LLMs.  Should we call out that Semantic Kernel is limited by the LLMs it's using, and those limitations include these?